website/docs/cli/commands/workspace/delete.mdx

```diff
@@ -13,20 +13,25 @@ Usage: `terraform workspace delete [OPTIONS] NAME [DIR]`
 
 This command will delete the specified workspace.
 
-To delete an workspace, it must already exist, it must have an empty state,
-and it must not be your current workspace. If the workspace state is not empty,
+To delete a workspace, it must already exist, it must not have resources under management,
+and it must not be your current workspace. If the workspace has resources under management,
```

Comment:
It turns out that even without considering Terraform Cloud workspaces, this is more correct because the requirement is that the state be empty of any resources (not outputs or metadata)

However, Terraform CLI seems to prefer the word "tracked" to "managed" today. I wonder if we should mirror that term here?

```suggestion
To delete a workspace, it must already exist, it must not be tracking resources,
and it must not be your current workspace. If the workspace is tracking resources,
```

![Screen Shot 2022-11-22 at 12 47 16 PM](https://user-images.githubusercontent.com/174332/203407463-50a34626-c5a3-4c8a-b92b-2a739aef8406.png)

Reply:
ah, this is an older draft! @JarrettSpiker product management let me know we're not saying RUM anymore, but more just "managed resources".  So I support removing!